addon/components/flash-message.js

```diff
@@ -73,6 +73,7 @@ export default Component.extend({
   willDestroy() {
     this._super();
     this._destroyFlashMessage();
+    run.cancel(this.pendingSet);
```

Comment:
Sure. I'll destructure `run.next` as well for consistency.

Again here using `get` and `set` is a little misleading to my eyes since it makes me expect that there is some observation occurring. I'll make the change and let me know if you want to switch it back.